integrations-and-supported-tools/prophet/scripts/Neptune_prophet.py

```diff
@@ -0,0 +1,32 @@
+import neptune.new as neptune
+import neptune_prophet.impl as npt_utils
```

Comment:
Is this how we want to do this? (Same question as for the ipynb example)

Reply:
No, it should be now importable as all the other integrations.

Reply:
I'm getting the following error
`ModuleNotFoundError: No module named 'neptune.new.integrations.prophet'`

When I  run 
`import neptune.new.integrations.prophet as npt_utils `

Reply:
OK, it seems that we didn't release the client version that has the integration added. Still, in the docs we probably want the proper way that would be available as soon as we release the current client.

Reply:
It's probably not released yet. :

Reply:
I'm still a bit confused. In the docs examples, which are based on the docstrings, there is no `import neptune.new.integrations.prophet as npt_utils`. Will it be needed or not? :)

Reply:
@normandy7 no, it will be importable like other integrations, so `from neptune.new.integrations.prophet import ...` , we just need to release the new client version that has the changes. In the docs we should use the "proper" way.